homeassistant/components/ezviz/__init__.py

```diff
@@ -70,7 +68,9 @@ async def async_setup_entry(hass, entry):
         _LOGGER.error("Unable to connect to Ezviz service: %s", str(error))
         raise ConfigEntryNotReady from error
 
-    coordinator = EzvizDataUpdateCoordinator(hass, api=ezviz_client)
+    coordinator = EzvizDataUpdateCoordinator(
+        hass, api=ezviz_client, api_timeout=entry.options[CONF_TIMEOUT]
```

Comment:
Is `CONF_TIMEOUT` not None if options was never set?

Reply:
Hi @milanmeu,

That's correct. On line L44 the code checks that options are set and set's them to the default if not set.